src/frontend/details/details.css

```diff
@@ -65,4 +65,7 @@
   .details-img, .details-image img {
     max-height: 41vh;
   }
+}
+p {
+color: white;
```

Comment:
Please indent this line :)

Reply:
this can be removed now, I was just using it to display my id when my pictures weren't loading.
removed.